english/heffalump/heffalump.md

```diff
@@ -0,0 +1,3 @@
+A heffalump you say? What's one of those...
+
+Well it's a magical beast with [super powers](../super-powers/practice-flying.md). Not to be confused with a [scarab](../scarab/goddess.md), or in fact a [marshmallow](../marshmallow.md).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.